### PR TITLE
ci: adds gcp tests

### PR DESCRIPTION
**Pull request by faiq**
**What problem does this PR solve?**:
adds gcp tests 
release images

**Which issue(s) does this PR fix?**:

* https://jira.d2iq.com/browse/D2IQ-NUMBER

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .github/workflows/gcp-e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Runs Azure tests when pull request opened, repopened or synchronized
+name: E2E Tests - Build Azure Image
```
Comment (faiq) [inline .github/workflows/gcp-e2e.yaml:2]:
```suggestion
name: E2E Tests - Build GCP Image
```

File: .github/workflows/release-images.yaml [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+        - "offline"
+        - "offline-fips"
+        - "offline-nvidia"
+        cloud_provider:
+        - "aws"
+        - "azure"
+        - "gcp"
+        - "ova"
+        exclude:
+        - os: "centos 7.9"
```
Comment (supershal) [inline .github/workflows/release-images.yaml:36]: this exclude list is crazy. 🧠 
let me try to check if `include` works better.

Comment (faiq) [reply .github/workflows/release-images.yaml]: moved to a new PR

**Review by supershal — APPROVED** [review]
The last e2e test migrated to GHA. 🎉

**Pull request merged**